recipes-extended/libcobalt/libcobalt_git.bb

```diff
@@ -23,6 +23,8 @@ COBALT_DEPENDENCIES ?= ""
 COBALT_DEPENDENCIES_brcm ?= "gst1-bcm"
 DEPENDS += "${COBALT_DEPENDENCIES}"
 
+RDEPENDS_${PN} += "gstreamer1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad wpeframework ${COBALT_DEPENDENCIES}"
```

Comment:
Aren't those dependencies also needed in the build time?

Reply:
yes, already in the DEPENDS flag